kv-components/vue/stories/KvSelect.stories.js

```diff
@@ -0,0 +1,107 @@
+import KvSelect from '../KvSelect.vue';
+
+export default {
+	title: 'KvSelect',
+	component: KvSelect,
+};
+
+const Template = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { KvSelect },
+	data: () => ({
+		exampleModel: 'example2',
+	}),
+	template: `
+		<div>
+			<label
+				for="example-id"
+				class="text-h4 block"
+			>
+				Choose an example
+			</label>
+			<kv-select
+				id="example-id"
+				:disabled="disabled"
+				@change="onChange"
+				v-model="exampleModel"
+				class="mt-2"
+			>
+					<option class="px-3 py-2" value="example1">Example one</option>
```

Comment:
The classes on the options are kind of annoying Because when using KvSelect, you'll have to remember to add these classes to your options elements. Could we possibly add these classes as mixins in the css of KvSelect? I'm thinking something like:
```
option {
  @apply px-3 py-2;
}

Reply:
Actually I think the `px-3 py-2` classes can be removed completely

Reply:
Thanks for catching this @eddieferrer, this was left over from me trying to style the open state of the dropdown before I realized that we're going with browser default and not what's shown in figma file.